### ISSUE TITLE
EPIC | Accessibility Audit Project

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#46756]
This epic consolidates all ticketed accessibility feedback  (find full list [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/accessibility/accessibility-audit-updates.md)) based on pre-existing open tickets and new tickets generated by the Accessibility team's recent audit.

#### Defect Level 1
- [x] #37363 Connected Apps/Content Heading (_Sprint 83_)
- [x] #46731 Multiple validation errors are not read out to screen-readers _(Update: Forms doesn't have bandwidth to address; we put in custom analytics to track this for a month)_ (defect-1)
- [x] once analytics are complete (see above) then will need new tickets for design and FE
- [x] #47646 Next Steps (research) _(Sprint 84)_

#### Defect Level 2
- [x] #46800 BAI _(Sprint 82)_
- [x] #46799 BAI _(Sprint 82)_
- [x] #47003 Screenreader (defect-2) (2 pts) _(related to #46614) (Sprint 83)_
- [x] #46614 Screenreader/DD _(Sprint 83)_
- [x] #46876 Design _(for #46614)_ _(Sprint 83)_
- [x] #46871 Discovery: move to one per page editing _(discussed 9/14)_ (related #47272 Address Validation)
- [x] #47136 Research Plan for one per page _(Sprint 83)_
- [x] #47378 Conduct Research _(Sprint 83)_
- [x] #48537 Address Validation (defect-3) and ~#43095~  Buttons that look like Links _(revisit once address validation approach decided)_ (defect-3) (Sprint 86)
- [x]  ~#48006 Bug: zoom with gesture disabled for modals on IOS~  <this is a sitewide issue and not unique to profile so we're not needed to cover this
- [ ] #40593 Sections of Profile are above the H1

#### Defect Level 3
- [x] #46801 BAI _(Sprint 82)_
- [x] #46802 BAI _(Sprint 82)_ 
- [x] #47266 Nametag - Discovery (defect-3) _(Sprint 84)_
- [x] #48263 Nametag - add Landmark-role
- [x] #46195 Aria Label to TTY 
- [x] #48753 Research Recommendations | Notifications Settings Spacing (2 pts)
- [x] #47299 Screenreader Accessibility Issue | Account Security (defect-3) (Sprint 86)
- [x] #49615 Account Security Part 2

#### Defect Level 4
- [x] #44534 Phone Number reads out as a numerical value (Sprint 82)
- [x] #46788 Content Updates for Modal (review with Content team)
- [x] #31449 Legend Elements (1 pt) (defect-4) (per Josh, this is being handles as a DST issue)
- [x] #46935 FE update for Modal (Are you sure?) (defect-4)
- [x] #49885 Profile | Accessibility | Mobile # (backlog)
- [x] #49589 Design | Profile | Accessibility | Mobile # (Sprint 87)

---

## Needs Grooming

- [x] #47721 Focus Area (defect-2) (see #46731 - addressed as part of this ticket) awaiting 30 days of metrics
- [x] #37041 Semantic Lists and headers (related to #19618) (defect-4)
- [ ]~#46609 Notify Settings/Mobile Number Read-out (defect-4) (will be done in  #49885)~
~- [x] #46854 Field Headings _(closed. being addressed in #19618)_~
- [x] #49652 Address Validation Improvements 

---

## On Hold

- [x] #40593 _Sections of Profile are above H1 (5 pts) _(deferred for now)__ (defect-2)

**Comment by SKasimow** [comment]
@ericasauv09 - could you do a check for the 508/Accessibility label and see if we've captured all the tickets in this "epic"? No rush - when you have time :)

**Comment by ericasauv09** [comment]
@SKasimow Do you mean just go through all tickets with that label in our backlog and make sure they are added to this list?

**Comment by SKasimow** [comment]
@ericasauv09 yes and any that might be in progress or closed. I caught a few and have added but good to have another set of eyes do a full sweep :)

**Comment by mtcA6** [comment]
The only remaining item for this ticket is : https://github.com/department-of-veterans-affairs/va.gov-team/issues/40593 which we're tracking via several other tickets. Closing this epic out to reduce the redundancy.
